src/js/api/config.js

```diff
@@ -126,7 +127,7 @@ const Config = function(options, persisted) {
             rates = rateControls;
         }
         rates = rates.filter(rate => isNumber(rate) && rate >= 0.25 && rate <= 4)
-            .map(rate => Math.round(rate * 4) / 4);
+            .map(rate => roundToHundredths(rate));
```

Comment:
Use math. This rounds to hundredths, and doesn't require converting the value to a string then back to a number.
```js
Math.round(rate * 100) / 100;
```

This will also always include hundredths. `toPrecision(2)` will not.